backoff/_sync.py

```diff
@@ -28,11 +28,9 @@ def retry_predicate(target, wait_gen, predicate,
 
     @functools.wraps(target)
     def retry(*args, **kwargs):
-
-        # update variables from outer function args
         nonlocal max_tries, max_time
```

Comment:
We should be able to remove this `nonlocal` statement now because we're no longer trying to update those values.

Reply:
Changed! 732eaa3